src/code/initSimulation.R

```diff
@@ -180,7 +180,19 @@ initSimulation <- function(XML = "", ParamList = numeric(0), whichInitParam = ""
 	{
 	  stop(.Call("RDCI_GetLastError"));
 	}
-
+	
+	# for the moment set all output variables/observer as persistable, so that their values are accessible after processSimulation
+	# this should be changed in the future to setting only otputs of interest  as persistable 
+	retval = .Call('RDCI_Invoke', DCI_Info$Handle, 'SetAllOutputsPersistable', "");
+	if (is.character(retval) & retval != "") 
+    {
+      stop(retval)
+    }
+	if (retval == 0)
```

Comment:
Most DCI functions return 1 for success and 0 for error. Actually it's a success flag what is being returned, not the error code.
Unless we don't want to change the whole DCI Core (and we don't :-), we have to accept this